pkg/cluster/storage.go

```diff
@@ -119,36 +117,39 @@ func (c *Cluster) storageBootstrap(ctx context.Context) error {
 // hashed with empty string and will check for any key that is hashed by different token than the one
 // passed to it, it will return error if it finds a key that is hashed with different token and will return
 // value if it finds the key hashed by passed token or empty string
-func (c *Cluster) getBootstrapKeyFromStorage(ctx context.Context, storageClient client.Client, token string) (*client.Value, bool, error) {
+func (c *Cluster) getBootstrapKeyFromStorage(ctx context.Context, storageClient client.Client, normalizedToken string) (*client.Value, error) {
 	emptyStringKey := storageKey("")
-	tokenKey := storageKey(token)
-
+	tokenKey := storageKey(normalizedToken)
 	bootstrapList, err := storageClient.List(ctx, "/bootstrap", 0)
 	if err != nil {
-		return nil, false, err
+		return nil, err
 	}
 	if len(bootstrapList) == 0 {
 		c.saveBootstrap = true
-		return nil, false, nil
+		return nil, nil
 	}
 	if len(bootstrapList) > 1 {
-		return nil, false, errors.New("found multiple bootstrap keys in storage")
+		logrus.Warn("found multiple bootstrap keys in storage")
```

Comment:
I think it is okay to error on multiple keys, we just need to ensure the keys contained in the list consist of only emptyStringKey and/or tokenKey.

Reply:
this can be prone to error, in that case we will going to need to delete the other key if found, what do you think?

Reply:
The purpose of returning an error before was to prevent multiple servers from using different sets of bootstrap credentials. In this case we know that we must remove the empty key, and the only valid remaining key should be the token key. I think it is ok to error if there is another bootstrap key that does not match our token, unless you can identify a case where that is a valid scenario. I am ok with keeping it a warning if it is a necessity but think we should be more selective about when we enter this code block.

Reply:
yeah thats not the case though, because we used to use the --token as it is, so if the server passed k10....:<token> it will use it as it is and not only the token part